game-engine/src/commonMain/kotlin/za/co/entelect/challenge/game/engine/renderer/WormsGameDetails.kt

```diff
@@ -30,4 +32,15 @@ class WormsGameDetails(config: GameConfig, wormsMap: WormsMap, player: WormsPlay
         return arrayMap.map { PrintableMapCell.buildForPerspectivePlayer(it, player) }
     }
 
+    private fun getOpponentsLastCommand(wormsMap: WormsMap, player: WormsPlayer): String {
+        val opponentFeedback = wormsMap
+            .getFeedback(currentRound - 1)
+            .filter { it.playerId != player.id }
+        return if (opponentFeedback.size == 0) {
+            "nothing"
+        } else {
+            opponentFeedback.get(0).command
```

Comment:
Please make sure that this behaves as expected when the opponent executes a select command in addition to another command.

Reply:
Yup.  I noticed that this morning and the fix is already in progress.